org_dependency_search.py

```diff
@@ -136,6 +136,8 @@ def run_query(org, repo, headers, url):
             raise Exception(
                 f"Query failed to run by returning code of" f" {request.status_code}. {query}"
             )
+        if "errors" in jsonified.keys():
```

Comment:
Do all errors return with a `errors` key even if there's only a single error?

Reply:
So, the errors field in the json is an array - I am only reporting the FIRST error - but "errors" is correct.

Reply:
And modified things to be better about declaring errors - probably looks better - value your opinion.

Reply:
Looks good, merge dat!